examples/fargate-serverless/main.tf

```diff
@@ -194,7 +194,7 @@ resource "null_resource" "modify_kube_dns" {
 
     # We are maintaing the existing kube-dns service and annotating it for Helm to assume control
     command = <<-EOT
-      echo "Setting implicit dependency on ${module.eks_blueprints.fargate_profiles["kube_system"].eks_fargate_profile_arn}"
```

Comment:
Thanks for the PR! However, this is correct as written - its referring to the Fargate profiles map input key which is `kube_system` https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/f5d31cf7a39e343f2c571e264dd0b7c13299abd3/examples/fargate-serverless/main.tf#L75